external-crates/move/crates/move-compiler/src/to_bytecode/translate.rs

```diff
@@ -1077,3 +1085,30 @@ fn binary_op(code: &mut IR::BytecodeBlock, sp!(loc, op_): BinOp) {
         },
     ));
 }
+
+fn abort_code(context: &mut Context, code: &mut IR::BytecodeBlock, ecode: H::Exp) {
+    use IR::Bytecode_ as B;
+    match ecode.exp {
+        sp!(cloc, H::UnannotatedExp_::Constant(nm)) => {
+            if let Some(error_code) = context.current_module().and_then(|m| {
+                context
+                    .constant_attributes(m, nm)
+                    .cloned()
+                    .and_then(|attrs| error_attribute_value(context.env, &attrs).map(|(_, v)| v))
+            }) {
+                let (_, (line_no, _), _) = context.env.file_mapping().location(cloc);
+                code.push(sp(
+                    cloc,
+                    B::ErrorConstant {
+                        line_number: line_no as u16,
```

Comment:
Should we check if `line_no` fits in `u16::MAX` and error (early-ish) if not?

Reply:
We clamp at u16::MAX, so if we exceed that value (...) we just say "no line number available" basically (just like with identifier, and constant value).